java-tests/validating/src/testFixtures/proto/spine/test/tools/validate/distinct.proto

```diff
@@ -36,7 +36,7 @@ option java_multiple_files = true;
 
 import "google/protobuf/any.proto";
 
-message ProtoSet {
-
-    repeated google.protobuf.Any element = 1 [(distinct) = true];
+message UniqueCollections {
```

Comment:
Please consider including a more complex use-case in testing. If I recall correctly, we also support the distinct map values (in addition to keys, which are unique by design). So let's test it with `Message`s as well.

The same applies the `repeated` field. We do support `Message`s, right?